## Deciphering gene regulatory mechanisms

Understanding gene regulatory mechanisms is crucial for uncovering the molecular basis of gene expression and its dysregulation in disease. Non-coding regulatory elements, such as enhancers and silencers, play key roles in controlling when and where genes are expressed, and mutations in these elements can lead to complex traits and diseases. I develop methods to decipher these mechanisms, focusing on non-coding regulatory elements, their interactions, and their impact on gene expression and disease mechanisms. By leveraging large-scale genomic and epigenomic datasets, such as those from ENCODE and ROADMAP, I systematically identify regulatory elements, including enhancers and silencers, and explore how genetic variants within these elements influence regulatory networks. I build comprehensive models of both bulk and singlecell level cell-type-specific regulatory interactions and gene expression by integrating both cis- and trans-regulatory networks. My work aims to bridge the gap between genetic variation and disease progression, providing a more nuanced understanding of how gene regulation is altered in disease states.

**Data Type**: ATAC-seq, ChIP-seq, RNA-seq, HiChIP, Hi-C, scRNA-seq, scATAC-seq 

